stac_fastapi/pgstac/tests/data/test_item.json

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
Should we build up a larger collection of test files in the future?

Reply:
yeah, and should probably use the same collection for multiple backends.